server/lib/src/server/migrations.rs

```diff
@@ -182,9 +176,6 @@ impl QueryServer {
         // we would have skipped the patch level fix which needs to have occurred *first*.
         if reload_required {
             write_txn.reload()?;
-            // We are not yet at the schema phase where reindexes will auto-trigger
-            // so if one was required, do it now.
-            write_txn.reindex(false)?;
```

Comment:
we no long er need to reindex?

Reply:
No, we don't :) I carefully checked this flow, and we don't need it.